locustfiles/poc.py

```diff
@@ -48,12 +44,10 @@ def _log_in(self):
     def _get_build_id(self):
         response = self.client.get(f'/a/{self.domain}/cloudcare/apps/v2/?option=apps', name='Web Apps apps')
         assert(response.status_code == 200)
-        build_id_map = {
-            app['copy_of']: app['_id']
-            for app in response.json()
-        }
-        assert(self.app_id in build_id_map)
-        self.build_id = build_id_map[self.app_id]
+        for app in response.json():
+            if app['copy_of'] == self.app_id:
+                return app['_id']
+        assert False, (f'"copy_of": "{self.app_id}" not found ', response.json())
```

Comment:
Nit: I think a more pythonic way of doing this assertion would be
```py
try:
    return next(app['_id'] for app in response.json() if app['copy_of'] == self.app_id)
except StopIteration:
    raise Exception(f'"copy_of": "{self.app_id}" not found ')
```
Or does Locust require an AssertionError to be raised?

Reply:
Oh, I like your version!

The assertion is just to help the user/developer figure out why something failed.